src/nix/fmt.cc

```diff
@@ -0,0 +1,53 @@
+#include "command.hh"
+#include "run.hh"
+
+using namespace nix;
+
+struct CmdFmt : SourceExprCommand {
+  std::vector<std::string> args;
+
+  CmdFmt() { expectArgs({.label = "args", .handler = {&args}}); }
+
+  std::string description() override {
+    return "reformat your code in the standard style";
+  }
```

Comment:
Speaking of formatting, please use the Nix formatting style :-)

E.g.

```suggestion
    std::string description() override 
    {
        return "reformat your code in the standard style";
    }
```